benefits/core/viewmodels.py

```diff
@@ -141,6 +142,17 @@ def __init__(self, **kwargs):
         self.button_text = kwargs.get("button_text")
 
 
+class AgencySelector(Modal):
+    """
+    Represents the agency selector modal dialog.
+    """
+
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+        agencies = kwargs.get("agencies", [])
```

Comment:
I think the logic of `models.TransitAgency.all_active()` should be here (maybe as the default) rather than in the view.

Reply:
Done in c1347be43cc4ee77dca0df4babe9f67b9174b595

Reply:
Sorry, had to rebase to fix a test -- moved the logic of `models.TransitAgency.all_active()` in a273d54a3ddd5021e2062943bce878338d4e7521